cmd/fleetctl/scripts.go

```diff
@@ -41,6 +41,15 @@ func runScriptCommand() *cli.Command {
 				return err
 			}
 
+			cfg, err := client.GetAppConfig()
+			if err != nil {
+				return err
+			}
```

Comment:
Shouldn't you make the run call and then process the error from the server?

Reply:
Wound up implementing some logic elsewhere to detect if I should show this error or not.

Reply:
Quick explanation since I reverted to my original approach: the designs specify that we should just show the "scripts are disabled" error, so we have to check that flag here before actually making the call. This way we avoid the "script running" output, which isn't included in the designs.